doc/man/sphinx-apidoc.rst

```diff
@@ -58,6 +58,10 @@ Options
 
    Maximum depth for the generated table of contents file.
 
+.. option:: --toc
```

Comment:
@shimizukawa which would you like `--toc` or `--tocfile`?

Reply:
`--tocfile` is better to me.

Reply:
Thank you for comment. done.